pubtools/pulplib/_impl/model/convert.py

```diff
@@ -32,4 +32,7 @@ def read_timestamp(value):
 
 
 def write_timestamp(value):
+    # value could be None if the report is empty
```

Comment:
I think it will be better to keep this method generic and move setting the value to current date in _export_dict()

Reply:
Hm, I did it here since the MaintenanceEntry object could also have the same problem, which has a timestamp, defaulted to None as well.

Reply:
That sounds fair. Since this is a generic method and not tied to maintenance_report only, please modify this comment to something like "defaults to current time if value is None"

Reply:
makes sense.